### PR TITLE
[MIG] crm_phonecall_timesheet: Migration to 10.0

### PR DESCRIPTION
**Pull request by chienandalu**
This module owns phonecall functionality from v8 crm_timesheet (https://github.com/OCA/hr-timesheet/tree/8.0/crm_timesheet) since phonecalls are now independent from the base and implemented in its proper OCA module.

cc @Tecnativa

### CODE REVIEW

File: crm_phonecall_timesheet/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# Copyright 2015 Javier Iniesta <javieria@antiun.com>
+# Copyright 2017 David Vidal <david.vidal@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': "CRM Timesheet",
+    'category': 'Customer Relationship Management',
+    'version': '10.0.1.0.0',
+    'depends': [
+        'crm',
```
Comment (pedrobaeza) [inline crm_phonecall_timesheet/__manifest__.py]: This one is implicit in the other dependencies

File: crm_phonecall_timesheet/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        'hr_timesheet',
+        'crm_timesheet',
```
Comment (pedrobaeza) [inline crm_phonecall_timesheet/__manifest__.py]: Why do you need this one?

File: crm_phonecall_timesheet/i18n/es.po [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+"POT-Creation-Date: 2015-08-31 10:18+0000\n"
+"PO-Revision-Date: 2015-08-31 10:18+0000\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: crm_timesheet
```
Comment (pedrobaeza) [inline crm_phonecall_timesheet/i18n/es.po]: This is not the correct module name

File: crm_phonecall_timesheet/views/crm_phonecall_view.xml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    <field name="inherit_id" ref="crm_phonecall.crm_case_phone_tree_view"/>
+    <field name="arch" type="xml">
+        <field name="date" position="after">
+            <field name="duration" widget="float_time" />
+        </field>
+        <field name="opportunity_id" position="after">
+            <button name="button_end_call"
+                    string="Stop"
+                    type="object"
+                    icon="fa-times-circle text-danger"
```
Comment (pedrobaeza) [inline crm_phonecall_timesheet/views/crm_phonecall_view.xml]: Don't put it on red and use `fa-stop-circle` the same as in project_timesheet_time_control.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: crm_phonecall_timesheet/__manifest__.py (hunk 2 above)
Comment (chienandalu) [reply crm_phonecall_timesheet/__manifest__.py]: You're right. It really doesn't

Comment (pedrobaeza) [reply crm_phonecall_timesheet/__manifest__.py]: Then remove it

**Comment by pedrobaeza** [comment]
Put this module auto_install=True

File: crm_phonecall_timesheet/__manifest__.py (hunk 2 above)
Comment (chienandalu) [reply crm_phonecall_timesheet/__manifest__.py]: It was removed on my commit: https://github.com/OCA/hr-timesheet/pull/106/commits/3da7180da6a150e97b9d598e80bc175c7dfdd891
But GitHub didn't outdate this...

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@chienandalu Please rebase now that the other module is merged.

@luismontalba please review

**Comment by pedrobaeza** [comment]
Please check Travis error.

**Comment by chienandalu** [comment]
@pedrobaeza @luismontalba All green now.

**Review by luismontalba — APPROVED** [review]
Tested on runbot

**Pull request merged**